hack/goreleaser.sh

```diff
@@ -114,16 +114,16 @@ build_single() {
   rm "$PWD/botkube"
 }
 
-build_test_single() {
+build_single_e2e(){
   export GORELEASER_CURRENT_TAG=v9.99.9-dev
   docker run --rm --privileged \
     -v "$PWD":/go/src/github.com/kubeshop/botkube \
     -v /var/run/docker.sock:/var/run/docker.sock \
     -w /go/src/github.com/kubeshop/botkube \
     -e GORELEASER_CURRENT_TAG=${GORELEASER_CURRENT_TAG} \
     goreleaser/goreleaser build --single-target --rm-dist --snapshot --id botkube-test -o "./botkube-e2e.test"
-  docker build -f "$PWD/build/test.Dockerfile" --platform "${IMAGE_PLATFORM}" -t "${IMAGE_REGISTRY}/${TEST_IMAGE_REPOSITORY}:${GORELEASER_CURRENT_TAG}" --build-arg TEST_NAME=botkube-e2e.test .
-  rm "$PWD/botkube-test"
+  docker build -f "$PWD/build/test.Dockerfile" --build-arg=TEST_NAME=botkube-e2e.test --platform "${IMAGE_PLATFORM}" -t "${TEST_IMAGE_REPOSITORY}:${GORELEASER_CURRENT_TAG}" .
```

Comment:
`IMAGE_REGISTRY` can be important since I may want to use other registries to maintain my images. Is there any reason to remove it?

Reply:
That's actually fixed - just not pushed yet.